### PR TITLE
[Doc] Add specific documentation on the available fw cmd controllers

### PR DESCRIPTION
**Pull request by christophfroehlich**
The existing documentation was honestly speaking rather misleading than helpful :) I hope this improved now

### CODE REVIEW

File: velocity_controllers/doc/userdoc.rst [hunk 1]
```diff
@@ -7,7 +7,41 @@ velocity_controllers
 
 This is a collection of controllers that work using the "velocity" joint command interface but may accept different joint-level commands at the controller level, e.g. controlling the velocity on a certain joint to achieve a set position.
 
-Hardware interface type
------------------------
+Currently, there is only one controller in this package:
```
Comment (saikishor) [inline velocity_controllers/doc/userdoc.rst:10]:
```suggestion
The package contains the following controllers:
```

File: position_controllers/doc/userdoc.rst [hunk 2]
```diff
@@ -7,7 +7,41 @@ position_controllers
 
 This is a collection of controllers that work using the "position" joint command interface but may accept different joint-level commands at the controller level, e.g. controlling the position on a certain joint to achieve a set velocity.
 
-Hardware interface type
------------------------
+Currently, there is only one controller in this package:
```
Comment (saikishor) [inline position_controllers/doc/userdoc.rst:10]:
```suggestion
The package contains the following controllers:
```

File: effort_controllers/doc/userdoc.rst [hunk 3]
```diff
@@ -7,7 +7,41 @@ effort_controllers
 
 This is a collection of controllers that work using the "effort" joint command interface but may accept different joint-level commands at the controller level, e.g. controlling the effort on a certain joint to achieve a set position.
 
-Hardware interface type
------------------------
+Currently, there is only one controller in this package:
```
Comment (saikishor) [inline effort_controllers/doc/userdoc.rst:10]:
```suggestion
The package contains the following controllers:
```

**Review by saikishor — APPROVED** [review]
LGTM. Thank you for the nice documentation.

**Review by bmagyar — APPROVED** [review]

**Pull request merged**